Code/GraphMol/FileParsers/MolFileParser.cpp

```diff
@@ -151,6 +151,17 @@ std::string getV3000Line(std::istream *inStream, unsigned int &line) {
   ++line;
   auto inl = getLine(inStream);
   std::string_view tempStr = inl;
+
+  // Reject any non-ascii characters.
```

Comment:
We should be careful with this.  Many SDF files have non-ascii characters (UTF-8 in Japan for example).   I'm not entirely sure if the character set for SDF files is actually specified.  But there are a lot of discussions on this:

https://forum.knime.com/t/sdf-writer-us-ascii-charset-limit/8707/20

Reply:
It appears all modern toolkits assume UTF-8 encoding.

Reply:
As @bp-kelley mentions: there are many, many SDF files and mol blocks out there that include non-ascii characters. We are not going to start rejecting those.

Reply:
Sorry about that - this is good to know. I've removed this for now. I'll try to find a more robust way of validating these files (trying to resolve https://osv.dev/vulnerability/OSV-2022-675)